RecoCTPPS/TotemRPLocal/interface/FastLineRecognition.h

```diff
@@ -84,11 +84,13 @@ class FastLineRecognition
       double Saw, Sbw, Sw, S1;
       double weight;
       double min_a, max_a, min_b, max_b;
+      bool firstCallToAdd = true;
```

Comment:
If you remove min_a, max_a, min_b, max_b, then you can also get rid of firstCallToAdd

Reply:
By the way: is there any reason for having all these class members and methods "protected" instead of "private"?

Reply:
I agree. If we delete min_a, max_a, min_b, max_b, then there is no reason for firstCallToAdd to exist. I would be happy to delete them all from the PR. Do you want me to do that?

In the repository nothing inherits from FastLineRecognition. Unless there is something outside the repository that I do not understand, there is no reason for these to be protected. Do you want me to change protected to private also?